libc/src/stdlib/freelist.h

```diff
@@ -99,7 +99,7 @@ bool FreeList<NUM_BUCKETS>::add_chunk(span<cpp::byte> chunk) {
 
   aliased.bytes = chunk.data();
 
-  unsigned short chunk_ptr = find_chunk_ptr_for_size(chunk.size(), false);
```

Comment:
These are part of #95065 but I think this should disappear after rebasing once that lands.